data/settings_mapping.json

```diff
@@ -234,6 +234,50 @@
         }
       ]
     },
+    {
+      "name": "starting_tab",
+      "text": "Starting Inventory",
+      "app_type": "generator",
+      "sections": [
+        {
+          "name": "starting_equipment_section",
+          "text": "Starting Equipment",
+          "col_span": 2,
```

Comment:
It is not particularly well explained, but setting col_span acts as an override for the default auto detection algorithm which does 1,2 or 4 columns depending on device width. Any reason in particular you don't want to have the 4 columns be used on the largest width?

Reply:
I was trying to keep the GUI from getting larger. I think there's an issue when using it over Xming though, because if I ran the GUI multiple times it would change sizes each time.

If there are better settings please let me know, or feel free to commit to this branch directly.

Reply:
That's weird. I will look into it